app/helpers/automated_tests_client_helper.rb

```diff
@@ -1,6 +1,4 @@
-require 'net/ssh'
-require 'net/scp'
-require 'json'
+require File.join(Rails.root, 'lib', 'automated_tests', 'server', 'automated_tests_server')

```

Comment:
This should probably be guarded by a `if MarkusConfigurator.markus_ate_server_host == 'localhost'`; this would make it possible, in principle, to omit the `AutomatedTestsServer` file from production, I think.